src/WebJobs.Script.WebHost/Management/LinuxSpecialization/IAssignmentHandler.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management.LinuxSpecialization
+{
+    public interface IAssignmentHandler
+    {
+        Task<string> ValidateContext(HostAssignmentContext assignmentContext);
```

Comment:
Do we follow `*Async` naming for task returning methods?

```suggestion
        Task<string> ValidateContextAsync(HostAssignmentContext assignmentContext);
```

Reply:
We do. Left the same comment, but backed out as it wouldn't be applicable if the recommendation is followed. I believe @Tonewall was just copying what was in `IInstanceManager`, which unfortunately, did not follow the convention.